src/Microsoft.AspNet.Identity/IIdentityKeyNormalizer.cs

```diff
@@ -4,15 +4,15 @@
 namespace Microsoft.AspNet.Identity
 {
     /// <summary>
-    ///     Used to normalize a user name
+    ///     Used to normalize keys for consistent lookups
     /// </summary>
-    public interface IUserNameNormalizer
+    public interface IIdentityKeyNormalizer
```

Comment:
Using just "key" to describe this concept seems to introduce some ambiguity, because there are already primary keys and foreign keys in the entities in addition to the properties we are normalizing. I like "lookup key" more, e.g. `IIdentityLookupKeyNormalizer`.